spec/index.spec.ts

```diff
@@ -21,11 +21,11 @@
 // SOFTWARE.
 
 import 'mocha';
-import { expect } from 'chai';
+import {expect} from 'chai';
```

Comment:
So, it looks like the rest of this repo has spaces around single element { objects } - for example, https://github.com/firebase/firebase-functions-test/blob/master/src/index.ts#L24.  Firebase-tools also keeps this style: https://github.com/firebase/firebase-tools/blob/master/src/commands/apps-list.ts#L5
I think your linter might have broken this throughout the files in this PR - can you switch this back?